profiles/models/profile.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+
+class Profile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+
+
+@receiver(post_save, sender=User)
+def create_user_profile(sender, instance, created, **kwargs):
+    if created:
+        Profile.objects.create(user=instance)
```

Comment:
Test to be added to validate the profile creation when user is created

Reply:
https://github.com/EliasBoulharts/squest/blob/billing_group_abstract_user/tests/test_models/test_profile.py#L12-L19